account/models.py

```diff
@@ -1,4 +1,5 @@
 from django.db import models
+from products.models import Product, Look 
```

Comment:
```suggestion
from products.models import (
    Product,
    Look
) 
```
두 개 이상의 클래스를 import 할 때에는 서로 다른 줄로 구분해주세요